android/quest/src/main/java/org/smartregister/fhircore/quest/ui/family/profile/FamilyProfileViewModel.kt

```diff
@@ -58,8 +59,13 @@ constructor(
     mutableStateOf(
       FamilyProfileUiState(
         overflowMenuItemConfigs =
-          overflowMenuFactory.retrieveOverflowMenuItems(OverflowMenuHost.FAMILY_PROFILE)
-      )
+          configurationRegistry.retrieveConfiguration<ProfileConfiguration>(
+              ConfigType.Profile,
+              "householdProfile" // TODO Pass profile id dynamically
+            )
+            .overFlowMenuItems
+        // overflowMenuFactory.retrieveOverflowMenuItems(OverflowMenuHost.FAMILY_PROFILE)
```

Comment:
what's up w/this line, do we need to keep it or can we remove entirely?

Reply:
An oversight. Should be removed.